context_windows.go

```diff
@@ -0,0 +1,9 @@
+package continuity
+
+import "os"
+
+// getUidGidFromFileInfo extracts the user and group IDs from a fileinfo.
+// This is Unix-specific functionality.
```

Comment:
The easiest way to encode an ACL is in SDDL which is essentially a string. See https://msdn.microsoft.com/en-us/library/windows/desktop/aa379567(v=vs.85).aspx. You can see an example of SDDL in the docker code base such as https://github.com/moby/moby/blob/master/pkg/system/filesys_windows.go#L99-L120. 

Calling them a `uid or gid` is really not appropriate on Windows.